modules/admin-credentials-root-secret-formats.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/manually-creating-iam.adoc
+
+ifeval::["{context}" == "manually-creating-iam-aws"]
+:aws:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-gcp"]
+:google-cloud-platform:
+endif::[]
+
+[id="admin-credentials-root-secret-formats_{context}"]
+= Admin credentials root secret format
+
+Each cloud provider uses a credentials root secret in the `kube-system`
+namespace by convention, which is then used to satisfy all `CredentialsRequests`
+and create their respective secrets. This is done either by minting new
+credentials, _mint mode_, or by copying the credentials root secret,
+_passthrough mode_.
+
+The format for the secret varies by cloud, and is also used for each
+`CredentialsRequest` secret.
+
+ifdef::aws[]
+
+.Amazon Web Services (AWS)
```

Comment:
Suggest putting "secret format" after the cloud provider name in each of these code block titles, e.g., "Amazon Web Services (AWS) secret format".